src/Storefront/Resources/app/storefront/src/scss/component/_line-item.scss

```
@@ -1,5 +1,11 @@
 .line-item {
-    margin-bottom: $spacer;
+    margin: 0;
+    padding: 10px 0;
```

Should we keep it `rem`?

```suggestion
    padding: 0.625rem 0;
```

---

I would be pro rem 👍